make the feed on html use the default delta

ryanlerch opened the pull request:
previously the link was hardcoded with a value of 127800. This commit
makes that link use the default delta value.

Signed-off-by: Ryan Lerch <rlerch@redhat.com>

abompard commented on datagrepper/templates/base.html:
```diff
@@ -38,7 +38,7 @@
         <ul class="nav navbar-nav nav-justified nav-underline">
             <li class="nav-item"><a class="nav-link" href="{{ url_for('index') }}">Getting&nbsp;started</a></li>
-            <li class="nav-item active"><a class="nav-link" href="{{ url_for('raw') }}?rows_per_page=1&delta=127800">Feed</a></li>
+            <li class="nav-item active"><a class="nav-link" href="{{ url_for('raw') }}?rows_per_page=1&delta={{ default_query_delta }}">Feed</a></li>
```
Shouldn't we just remove the `delta` argument?

ryanlerch replied:
i have a feeling there was a reason for doing this -- but cant remember why now.

abompard commented:
Hey @ryanlerch , could you rebase please? Also I had a small comment.

ryanlerch committed:
remove hardcoded delta value on raw feed URL

previously the link was hardcoded with a value of 127800. This commit
removes that value

Signed-off-by: Ryan Lerch <rlerch@redhat.com>
